templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -75,7 +68,7 @@ contents:
       for file in "${files[@]}"; do
         file="$(basename $file)"
         # Also remove files in underlay
-        for path in "${NM_CONN_PATH}" "${NM_CONN_UNDERLAY}"; do
+        for path in "${NM_CONN_PATH}"; do
```

Comment:
Maybe remove the for and rewrite the `# Also remove files in underlay` comment?

Reply:
done. thanks!